### ISSUE TITLE
NRF52811 support

### ISSUE THREAD

**Issue by vvzvlad** [RIOT-OS/RIOT#13054]
#### Description
I have a developed device with NRF52811 that I would like to run under RIOT OS. As far as I can see, there is no support for 52811 in RIOT now. 
What do I have to do to get it to appear? What prevents it from appearing? 
I plan to use only 802.15.4 radio without BLE AoA, AoD and LR.

The zephyr project has support for this processor (https://docs.zephyrproject.org/latest/boards/arm/nrf52811_pca10056/doc/index.html), and it seems to me that adding support to RIOT will not be very difficult: https://github.com/zephyrproject-rtos/zephyr/tree/master/soc/arm/nordic_nrf/nrf52

#### Links
https://www.nordicsemi.com/Products/Low-power-short-range-wireless/nRF52811

**Comment by benpicco** [comment]
Might be as simple as adding

```make
ifneq (,$(filter nrf52811xxaa,$(CPU_MODEL)))
  ROM_LEN ?= 0x30000
  RAM_LEN ?= 0x6000
endif
```

to `cpu/nrf52/Makefile.include` - did you try that aleady?

**Comment by Citrullin** [comment]
@benpicco Do I get this right? The `CPU_IRQ_NUMOF` is the number of NVICs?

**Comment by benpicco** [comment]
Yes, you probably also have to adapt `cpu/nrf52/vectors.c`

**Comment by benpicco** [comment]
Fixed by #14008
